number-is-nan/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for number-is-nan 1.0
+// Project: https://github.com/sindresorhus/number-is-nan#readme
+// Definitions by: Mohamed Hegazy <https://github.com/mhegazy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = number_is_nan;
+declare function number_is_nan(p0: any): boolean;
```

Comment:
I wonder if this should be `p0: number`? Same for `isFinite`. If the value you are passing in is not a number, it won't crash at runtime, but you will get `false` every time, so it isn't doing what you intend. So it's like passing in an extra argument to a function: seems too loose to allow it.